Player/Player.gd

```diff
@@ -0,0 +1,26 @@
+extends CharacterBody2D
+
+
+const speed = 200
+@onready var anim = get_node("AnimatedSprite2D")
+
+func _ready():
+	anim.play("Idle")
+
+func _process(delta):
+	var velocity = Vector2()
+	if Input.is_action_pressed("move_right"):
+		velocity.x += 1
+	if Input.is_action_pressed("move_left"):
+		velocity.x -= 1
```

Comment:
Since you are already updating the animation, you can flit the player to:

```suggestion
	if Input.is_action_pressed("move_right"):
		_velocity.x += 1
		anim.flip_h = false
	if Input.is_action_pressed("move_left"):
		_velocity.x -= 1
		anim.flip_h = true
```